Tasks/DockerV0/task.loc.json

```diff
@@ -341,10 +341,10 @@
   ],
   "instanceNameFormat": "ms-resource:loc.instanceNameFormat",
   "execution": {
-    "Node10": {
```

Comment:
Nothing changes - what's the purpose of this?

Reply:
I'm putting Node10 below Node16 as part of msal-node upgrade